src/vconsole/vconsole-setup.c

```diff
@@ -610,14 +621,31 @@ static int run(int argc, char **argv) {
 
         umask(0022);
 
-        if (argv[1])
+        if (argv[1]) {
                 fd = verify_source_vc(&vc, argv[1]);
-        else
+                if (fd < 0)
+                        return EXIT_FAILURE;
+        } else {
                 fd = find_source_vc(&vc, &idx);
-        if (fd < 0)
-                return fd;
+                if (fd < 0) {
+                        if (fd != -EBUSY) {
+                                log_error_errno(fd, "No usable source console found: %m");
+                                return EXIT_FAILURE;
+                        }
+                        exit_early = true;
```

Comment:
if you want to add such a boolean, that's ok, but please name it after when it is set, not about its effect. i.e. "vc_unavailable" or so

Reply:
but frankly, i'd not add the boolean var, and just check if fd >= 0

Reply:
you meant `if (fd == -EBUSY)` instead of `if (exit_early)`, right ?

Reply:
well, fd < 0…